src/index.html

```diff
@@ -6,8 +6,49 @@
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <title>Product cards</title>
     <link rel="stylesheet" href="./styles/index.scss">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Product cards</h1>
+    <div class="card" data-qa="card">
+      <a class="card__showcase" href="#">
+        <img
+          class="card__photo"
+          src="./images/imac.jpeg"
+          alt="APPLE A1419 iMac 27 product photo"
+        >
+      </a>
+
+      <div class="card__description">
+        <h2 class="card__productName">APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</h2>
+
+        <p class="card__productCod">Product code: 195434</p>
+
+        <div class="card__marks">
+          <div class="stars">
```

Comment:
Active stars should change depending on the modifier. For example, if I add the `stars--3` class it should be 3 active stars